### PR TITLE
Expose Listener close function

### PR DESCRIPTION
**Pull request by panpur**
## Description
Close listener on drop

## Testing
Manual test

## Documentation
No

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1487,6 +1491,12 @@ impl Listener {
             panic!("ListenerStart failed, {:x}!\n", status);
         }
     }
+
+    pub fn close(&self) {
```
Comment (nibanks) [inline src/lib.rs:1495]: Instead of this, I think you should just implement `Drop for Listener`.

File: src/lib.rs [hunk 2]
```diff
@@ -1198,6 +1198,10 @@ impl Api {
         Api { table: new_table }
     }
 
+    pub fn close(&mut self) {
```
Comment (nibanks) [inline src/lib.rs:1201]: Why does `close` need to be explicitly exposed?

Comment (panpur) [reply src/lib.rs]: I thought to add it since Connection & Stream also have explicit close, but anyway I've removed it.

File: src/lib.rs (hunk 1 above)
Comment (panpur) [reply src/lib.rs]: I've moved it to drop function.

**Review by nibanks — APPROVED** [review]

**Pull request merged**